crates/block-producer/src/main.rs

```diff
@@ -1,3 +1,10 @@
+#[cfg(not(target_env = "msvc"))]
+use tikv_jemallocator::Jemalloc;
+
+#[cfg(not(target_env = "msvc"))]
```

Comment:
```suggestion
#[cfg(all(not(target_env = "msvc"), not(target_os = "macos")))]
```

https://github.com/nervosnetwork/ckb/blob/b3da297eda0b4bea7892b12124c783ce42bd701f/src/main.rs#L5